spec/02-integration/20-wasm/01-admin-api_spec.lua

```diff
@@ -387,4 +388,162 @@ describe("WASMX admin API [#" .. strategy .. "]", function()
 
 end)
 
+describe("WASMX admin API - wasm = off [#" .. strategy .. "]", function()
+  local admin
+  local bp, db
+  local service
+
+  lazy_setup(function()
+    bp, db = helpers.get_db_utils(strategy, {
+      "routes",
+      "services",
+    })
+
+    service = assert(db.services:insert {
+      name = "wasm-test",
+      url = "http://wasm.test",
+    })
+
+    assert(helpers.start_kong({
+      database = strategy,
+      nginx_conf = "spec/fixtures/custom_nginx.template",
+      wasm = "off",
+    }))
+
+    admin = helpers.admin_client()
+  end)
+
```

Comment:
Looks as though you're missing a `lazy_teardown()` to close the admin client and stop kong. I think this is why one of the other tests is failing due to not being able to start kong.

Reply:
Yep, good catch!